doc/modules/changes/20250129_jdannberg

```diff
@@ -0,0 +1,6 @@
+Changed: ASPECT now re-generates particles in each adaptive
+refinement cycle instead of only once after global refinement.
+This means that particle locations during adaptive refinement
```

Comment:
Can you modify this to make clear this only refers to *initial* adaptive and global refinement cycles here or in the second sentence? At the moment it sounds like we are regenerating particles during every adaptive refinement.